MvvmCross.Forms/MvxFormsLog.cs

```diff
@@ -4,8 +4,12 @@
 
 using System.Runtime.CompilerServices;
 using MvvmCross.Logging;
+using Xamarin.Forms;
 
 [assembly: InternalsVisibleTo("MvvmCross.Forms.Platforms.Wpf")]
+[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Bindings))]
```

Comment:
A little bit of a weird place to put these. I would probably still create a AssemblyInfo.cs file and just not write anything else than:

```csharp
using Xamarin.Forms;

[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Bindings))]
[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Converters))]
[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Views))]
```

No namespace or class definition and it should be good.